src/components/common/program-section-templates/single-image-bottom/SingleImageBottom.module.scss

```
@@ -1,5 +1,5 @@
 @import '../shared/BaseSectionTemplate.module.scss';
-$img-width: 1440px;
+$img-width: 1330px;
```

Stylelint (17.5.0)

[error] Expected an empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)